src/YarpUtilities/src/Helper.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file Helper.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2019 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <BipedalLocomotionControllers/YarpUtilities/Helper.h>
+
+using namespace BipedalLocomotionControllers;
+bool YarpUtilities::addVectorOfStringToProperty(yarp::os::Property& prop,
+                                                const std::string& key,
+                                                const std::vector<std::string>& list)
+{
+    // check if the key already exists
+    if (prop.check(key))
+    {
+        std::cerr << "[YarpUtilities::addVectorOfStringToProperty::addVectorOfStringToProperty] "
+                     "The property already exist."
+                  << std::endl;
+        return false;
+    }
+
+    prop.addGroup(key);
+    yarp::os::Bottle& bot = prop.findGroup(key).addList();
```

Comment:
``prop.addGroup(key)`` already returns a ``Bottle``. You can merge the two lines in
```cpp
yarp::os::Bottle& bot = prop.addGroup(key).addList();
```

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-controllers/pull/5/commits/2bc1f2b480dc4f14e23860085b9413eff74bfc69